test/cbmc/proofs/DHCP/DHCPProcess/Makefile.json

```diff
@@ -36,10 +36,10 @@
   # The number of times GetNetworkBufferWithDescriptor can be allowed to fail
```

Comment:
Comments are not supported in json file

Reply:
We can remove the copyright, but in my opinion its better to keep these comments as its helpful for future maintenance and the python script is taking care of the comments for now.

Reply:
Yeah we can skip for existing comments